MSET9_installer_script/mset9.bat

```diff
@@ -1,12 +1,34 @@
-@echo off
-chcp 65001 > nul
-py -V > nul
-if %errorlevel% NEQ 0 (
-    echo Python 3 is not installed.
-    echo Please install Python 3 and try again.
-    echo https://www.python.org/downloads/
-    echo.
-    pause
-    exit
-)
-py -3 mset9.py
\ No newline at end of file
+@echo off
+chcp 65001 > nul
+
+py -V 2> nul
+if %errorlevel% EQU 0 py -3 mset9.py
+python -V 2> nul
+if %errorlevel% EQU 0 python -3 mset9.py else (

```

Comment:
This will execute even if “py -3 mset9.py” was already successful. In your above if statement you should also exit the script, or you should otherwise rearrange this logic